build_image.sh

```diff
@@ -15,14 +15,14 @@ fi
 setup_type=$1
 app_path=$PWD
 app_name=${app_path##*/}
-parent_path=`dirname $PWD`
+#parent_path=`dirname $PWD`
 
 echo "-- Generating image for $setup_type" container.
```

Comment:
I know this isn't you - but dont understand why the " is not at the end of the line :-)

Reply:
will update.